### PR TITLE
feat: :sparkles: Added a new hook to post to slack on every comment

### PR DESCRIPTION
**Pull request by rebelchris**
I've added a new slack hook to act on every comment.
For now this will post in the same channel but with out avo green color.

I haven't been able to test end-to-end because I don't know the new webhook url.

The test and local CDC all go well though.

DD-326 #done

### CODE REVIEW

**Review by idoshamun — CHANGES_REQUESTED** [review]
The best practice here is not to use CDC but rather create a new subscription that only sends the slack notification. The reason is that we want to optimally run exactly one side-effect for every message we handle. Otherwise, the retry mechanism can be a mess. Imagine you sent the slack notification here and then for some reason something failed down the line. It will cause Pub/Sub to send again the same message and basically flood our Slack channel. This is something called distributed transaction in distributed systems. The best way to handle it, is to bypass the need for such transactions by introducing multiple subscriptions that listens to the same domain event. In our case the domain event is pos commented. The CDC topic is not a domain event but rather something technical that we use to generate domain events out of it.

**Comment by rebelchris** [comment]
@idoshamun Thanks, that makes sense, I'll refactor to it'w own subscription.
I did see the report webhook runs over the CDC, that's still fine then?

**Comment by rebelchris** [comment]
@idoshamun Refactored to two new subscriptions one for the commentComment and one for postComment.
It seems you cannot re-use the same subscription on the same topic, hence I made two.

Looking forward to hearing what you think.

File: infra/workers.ts [hunk 1]
```diff
@@ -58,6 +58,10 @@ export const workers: Worker[] = [
     topic: 'comment-commented',
     subscription: 'comment-commented-author-mail',
   },
+  {
+    topic: 'post-commented',
```
Comment (idoshamun) [inline infra/workers.ts:62]: 👀 

```suggestion
    topic: 'comment-commented',
```

File: infra/workers.ts [hunk 2]
```diff
@@ -58,6 +58,10 @@ export const workers: Worker[] = [
...
+    subscription: 'comment-comment-slack-message',
```
Comment (idoshamun) [inline infra/workers.ts:63]:
```suggestion
    subscription: 'comment-commented-slack-message',
```

File: infra/workers.ts [hunk 3]
```diff
@@ -86,6 +90,10 @@ export const workers: Worker[] = [
     topic: 'post-commented',
     subscription: 'post-commented-redis',
   },
+  {
+    topic: 'post-commented',
+    subscription: 'post-comment-slack-message',
```
Comment (idoshamun) [inline infra/workers.ts:95]:
```suggestion
    subscription: 'post-commented-slack-message',
```

File: src/workers/commentCommentSlackMessage.ts [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+import { notifyNewComment } from '../common';
+
+interface Data {
+  userId: string;
+  childCommentId: string;
+  postId: string;
+}
+
+const worker: Worker = {
+  subscription: 'comment-comment-slack-message',
```
Comment (idoshamun) [inline src/workers/commentCommentSlackMessage.ts:12]:
```suggestion
  subscription: 'comment-commented-slack-message',
```

File: src/workers/postCommentSlackMessage.ts [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+import { notifyNewComment } from '../common';
+
+interface Data {
+  userId: string;
+  commentId: string;
+  postId: string;
+}
+
+const worker: Worker = {
+  subscription: 'post-comment-slack-message',
```
Comment (idoshamun) [inline src/workers/postCommentSlackMessage.ts:12]:
```suggestion
  subscription: 'post-commented-slack-message',
```

**Review by idoshamun — CHANGES_REQUESTED** [review]
Other than naming concerns I'm speechless.
Way to go :)

**Review by idoshamun — APPROVED** [review]
Looks great!

**Review by sshanzel — APPROVED** [review]
So cool to see we can now confidently say the CDC pipeline works. Well done! 🚢

**Pull request merged**